src/main/resources/db.changelog/changes/floor.yaml

```diff
@@ -0,0 +1,52 @@
+databaseChangeLog:
+  - logicalFilePath: no-path
+  - changeSet:
+      id: floor teble init
+      author: saidamin
+      preConditions:
+        onFail: MARK_RAN
+        not:
+          tableExists:
+            schemaName:
+            tableName: floor
+      changes:
+        - createTable:
+            tableName: floor
+            columns:
+              - column:
+                  name: fl_id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    primaryKeyName: pk_fl_id
+              - column:
+                  name: of_id
+                  type: int
+                  constraints:
+#                    foreignKeyName: fk_of_id
```

Comment:
why it commented?

Reply:
sorry when I created this changelog I didn't have office table than I had to comment this part of code 
now I fixed it